src/main/java/yahav/photomosaic/ImagePixels.java

```diff
@@ -96,12 +97,13 @@ public double colorDistance(Color firstColor, Color secondColor){
     }
     public void closestColorDiff(){
```

Comment:
What does this method do?

Reply:
this method is supposed to find the closest matching average color between the input image and source images